src/app/books/book-details/book-details.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
This PR is for List Books. So avoid adding the other feature files in here.

Reply:
right